src/main.rs

```diff
@@ -34,20 +34,26 @@ async fn main() -> Result<(), Box<dyn Error>> {
     // Set SENTRY_DSN environment variable to enable Sentry.
     // Avoid its default reqwest transport for now due to issues w/
     // likely grpcio's boringssl
+    // Commented out with sentry 0.19
```

Comment:
By the way, we might be impacted here by the bug I found with sentry 0.19:
https://github.com/getsentry/sentry-rust/issues/237

TL;DR workaround is to either use the `debug-logs` feature or call `Hub::with` on the main thread with a no-op before doing any sentry things.